data/mods/blazemod/blaze_test.json

```diff
@@ -235,7 +235,7 @@
         "price" : 1500000,
 		"watertight": true,
 		"seals": true,
-		"contains": 0
+		"contains": 1
```

Comment:
> The author is maintaining a separate tree that he isn't PRing.

> get rid of the mainlined version of the mod to lose the responsibility to maintain a very outdated version of the mod.

Given the above the only logical outcome is the latter. Unmaintained code just bit rots.